content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -48,82 +48,99 @@ For a full list of supported libraries and processor architectures, see [Compati
 
 ## Installation and getting started
 
-### Automatic instrumentation
-
 <div class="alert alert-warning">
-  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution in your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution in your application environment.</li></ul>
 </div>
 
-#### Installing the tracer
+### Installation overview
+
+1. [Install the tracer][8]
+2. [Enable the tracer for your service][9]
+3. [Configure the Datadog Agent for APM][10]
+4. [View your live data][11]
+
+### Install the tracer
+
+The Datadog .NET Tracer can be installed machine-wide to allow services across an entire machine to be instrumented, or it can be installed on a per-application-basis to allow developers to manage the instrumentation via the application’s dependencies. To see machine-wide installation options, view the Windows or Linux tab. To see per-application installation options, see the NuGet tab.
 
 {{< tabs >}}
 
 {{% tab "Windows" %}}
 
+To install the .NET Tracer:
+
 1. Download the [.NET Tracer MSI installer][1]. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
 
 2. Run the .NET Tracer MSI installer with administrator privileges.
 
-3. Enable instrumentation in your service by setting the required environment variables. See the section *Instrumenting your service*, below.
 
-4. Create application load.
+[1]: https://github.com/DataDog/dd-trace-dotnet/releases
+{{% /tab %}}
+
+{{% tab "Linux" %}}
+
+To install the .NET Tracer:
 
-5. Visit [APM Live Traces][2].
+1. Download the latest [.NET Tracer package][1] that supports your operating system and architecture.
+
+2. Once downloaded, install the package with one of the commands below:
+
+```bash
+# For Debian or Ubuntu, install the DEB package
+sudo dpkg -i ./datadog-dotnet-apm_<TRACER_VERSION>_amd64.deb
+/opt/datadog/createLogPath.sh # Creates the .NET Tracer log directory with the appropriate permissions at /var/log/datadog/dotnet
```

Comment:
I would probably make these a one-liner (as more likely for people to copy it correctly). What do you think? e.g.
```suggestion
# and create the .NET Tracer log directory with the appropriate permissions at /var/log/datadog/dotnet
sudo dpkg -i ./datadog-dotnet-apm_<TRACER_VERSION>_amd64.deb && /opt/datadog/createLogPath.sh 
```

Reply:
Yeah this is a good idea. Done in the latest commits